site_scons/env_modules.py

```diff
@@ -56,7 +57,8 @@ def _module_func(self, command, *arguments):  # pylint: disable=no-self-use
 
         # pylint: disable=consider-using-with
         try:
-            print(f"Going to run {cmd}")
+            if not self._silent:
+                print(' '.join(cmd))
```

Comment:
Maybe define a `self.print` so we don't need this check everywhere?

Reply:
I considered it but there are only two calls.